source/News.cpp

```diff
@@ -83,14 +96,27 @@ void News::Load(const DataNode &node, const ConditionsStore *playerConditions)
 		else if(tag == "message")
 		{
 			if(remove)
+			{
 				messages = Phrase{};
+				if(hasValue || child.HasChildren())
+					child.PrintTrace("Removing all messages; removal of single messages is not supported:");
+			}
 			else
 				messages.Load(child);
 		}
 		else if(tag == "to" && hasValue && child.Token(valueIndex) == "show")
 		{
+			if(add)
+			{
+				child.PrintTrace("Skipping 'add to show'; adding to conditions is not supported:");
+				continue;
+			}
 			if(remove)
+			{
 				toShow = ConditionSet{};
+				if(hasValue || child.HasChildren())
```

Comment:
`hasValue` is always true at this point.
```suggestion
				if(child.HasChildren())
```
The rest of the nodes you added this check for don't ever read their value, only their children, so we should probably remove the check from those too.